### PR TITLE
fozzie-components@v4.1.0 – Cache Component in Circle CI #trivial

### PR DESCRIPTION
**Pull request by jamieomaguire**
Adds the new f-restaurant-card component to the cached components list in Circle CI

Also updates the basic component test to use the newer patterns other molecules use.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+Latest (to be added to next release)
```
Comment (ashleynolan) [inline CHANGELOG.md]:
```suggestion
v4.1.0
```

Just as we don't 'release' the root, we just version changes for documentation purposes

Comment (ashleynolan) [reply CHANGELOG.md]: You'll need to bump the package.json version number too

Comment (jamieomaguire) [reply CHANGELOG.md]: ah, maybe I misunderstood but I thought this is what @dandel10n was asking me to do! Will update to the suggestion :)

**Review by ashleynolan — DISMISSED** [review]

**Review by AlixRichards — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dandel10n) [reply CHANGELOG.md]: I thought that we don't need to bump the version for this kind of change...

Comment (ashleynolan) [reply CHANGELOG.md]: For changes to the root config, I bump on any change (as it's not being released anyway – it's just to catalog the change).

But for small changes in a releasable package that don't actually need to be released, then we don't need to bump the version every time.

**Review by dawidchar — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**